bin/start-backend

```diff
@@ -9,4 +9,7 @@ export CLICKHOUSE_API_PASSWORD="apipass"
 export CLICKHOUSE_APP_USER="app"
 export CLICKHOUSE_APP_PASSWORD="apppass"
 
-uvicorn --reload posthog.asgi:application --host 127.0.0.1 --log-level debug --reload-include "posthog/" --reload-include "ee/" --reload-include "products/"
+# Suppress frozen modules warning since we only debug our code, not stdlib
+export PYDEVD_DISABLE_FILE_VALIDATION=1
+
+python ${DEBUG:+ -m debugpy --listen 127.0.0.1:5678} -m uvicorn --reload posthog.asgi:application --host 127.0.0.1 --log-level debug --reload-include "posthog/" --reload-include "ee/" --reload-include "products/"
```

Comment:
Hmm, we have DEBUG=1 set in a lot of places, are you intentionally planning on reusing that env?